src/main.rs

```diff
@@ -34,12 +34,14 @@ use structopt::StructOpt;
 mod config;
 mod models;
 
+/// Define the CLI parameters
 #[derive(StructOpt)]
 struct Cli {
     /// A path to a simulation config
     config: String,
 }
 
+/// In the beginning, the world was public static void main…
```

Comment:
And it was not good.

Reply:
But it was Java!

Reply:
This is the way the world ends, not with an int but with a NullPointerException